python/setup.py

```diff
@@ -184,7 +184,6 @@ def get_packages(self):
 # bindings are created.
 generated_python_directories = [
     "ray/core/generated",
-    "ray/core/generated/experimental",
```

Comment:
This reverts https://github.com/ray-project/ray/pull/37403 since it's no longer needed.